widgetssdk/src/main/java/com/glia/widgets/core/fileupload/domain/AddFileToAttachmentAndUploadUseCase.java

```diff
@@ -46,7 +44,7 @@ private void onHasOngoingEngagement(FileAttachment file, Listener listener) {
         if (isSupportedFileCountExceeded()) {
             fileAttachmentRepository.setSupportedFileAttachmentCountExceeded(file.getUri());
             listener.onError(new SupportedFileCountExceededException());
-        } else if (isSupportedFileSizeExceeded(file)) {
+        } else if (isSupportedFileSizeLargerThen25MB(file)) {
```

Comment:
Potentially could change

Reply:
removed one useless method, didn't rename. But I'll rename it to be cleaner.